config/config.py

```diff
@@ -64,12 +64,21 @@ def build_config(
                 }
             ),
             "PU_reweighting_hist": "pileup",
-            "met_filters": ["Flag_goodVertices", "Flag_METFilters"],
+            "met_filters": [               
```

Comment:
is this in line with the JetMET recommenations for filters ?

add a modification rule for the 2017 / 2018 only Filter, also looking into our non-UL configs, `Flag_eeBadScFilter` was only running for data and embedded, but not for mc samples

Reply:
Yes, these are the recommended filters. Actually, it is also recommended not to use the `Flag_METFilters` for nanoAOD.
See more [here](https://twiki.cern.ch/twiki/bin/viewauth/CMS/MissingETOptionalFiltersRun2#Analysis_Recommendations_for_ana)

Reply:
Your are right about `Flag_eeBadScFilter` for non-UL, but it is recommended now for 17/18 UL mc samples .

Reply:
Done